contrib/utilities/update_all_files.sh

```diff
@@ -5,16 +5,17 @@
 # effect, because all files should be on the current version, but if you change
 # the code, or add new files it can be handy to execute this script.
 
-DOC_DIR=`dirname $0`
-BASE_DIR=`pwd $DOC_DIR/..`
```

Comment:
I don't understand how this could have worked. ``pwd`` does not take any arguments.